### PR TITLE
Update run instructions in README.md

### PR DESCRIPTION
**Pull request by jeannielynnmoulton**
Resolves https://github.com/NTD-Modelling-Consortium/endgame-project/issues/40

This updates the run instructions in the README and adds some notes about problems one might encounter with different python versions.

Limitation: I did not test this on a fresh machine and do not have an easy way to. If the reviewer has a good way to test on a fresh machine, that would be great!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,19 +39,38 @@ Numerous different examples are included in the Jupyter notebook `trachoma_tests
 
 ### How to run
 
-Install [pipenv](https://drive.google.com/drive/folders/1Or6lUkymYd_p031xKGZLcnTV4GYf-oYb) according to the instructions for your OS.
+Install `pipenv` and `setuptools` according to the instructions for your OS, e.g. 
+```commandline
+pip install pipenv
+pip install setuptools
```
Comment (tlestang) [inline README.md:45]: I don't think setuptools is needed.

File: README.md [hunk 2]
```diff
@@ -39,19 +39,38 @@ Numerous different examples are included in the Jupyter notebook `trachoma_tests
 
 ### How to run
 
-Install [pipenv](https://drive.google.com/drive/folders/1Or6lUkymYd_p031xKGZLcnTV4GYf-oYb) according to the instructions for your OS.
+Install `pipenv` and `setuptools` according to the instructions for your OS, e.g. 
+```commandline
+pip install pipenv
```
Comment (tlestang) [inline README.md:44]: We should probably recommend against installing `pipenv` wthin the system or global python env.

Users could `pipx install pipenv` (using [pipx](https://pipx.pypa.io/stable/)), create a venv themselves prior to installing pipenv or at the very leat `pip install --user pipenv`

**Comment by tlestang** [comment]
`pipenv install .` ran fine on my machine with python 3.8.13.

However it looks like there is missing data for `simple_example.py`:

```shell
# From the project root
pipenv run python simple_example.py 
Traceback (most recent call last):
  File "/mnt/shared/scratch/repos/ntd-model-trachoma/simple_example.py", line 72, in <module>
    pickleData =  pickle.load(open(PickleFilePath, 'rb'))
FileNotFoundError: [Errno 2] No such file or directory: 'OutputVals_BDI06375.p'
```

**Comment by jeannielynnmoulton** [comment]
> `pipenv install .` ran fine on my machine with python 3.8.13.
> 
> However it looks like there is missing data for `simple_example.py`:
> 
> ```shell
> # From the project root
> pipenv run python simple_example.py 
> Traceback (most recent call last):
>   File "/mnt/shared/scratch/repos/ntd-model-trachoma/simple_example.py", line 72, in <module>
>     pickleData =  pickle.load(open(PickleFilePath, 'rb'))
> FileNotFoundError: [Errno 2] No such file or directory: 'OutputVals_BDI06375.p'
> ```

Thanks for the review. This is the note "At this point you will need to add files to the root of the project directory. Please ask your administrator for these files." They are not stored in the repo.

File: README.md [hunk 3]
```diff
@@ -39,19 +39,38 @@ Numerous different examples are included in the Jupyter notebook `trachoma_tests
...
+At this point you will need to add files to the root of the project directory. Please ask your 
+administrator for these files.
 
+Then you can run an example file using `pipenv`:
+```commandline
+pipenv run python simple_example.py
 ```
-	$ pipenv install . # sets up per-project python environment ('env')
-	$ pipenv shell # starts a per-project shell using that env
-	(ntd-model-trachoma) $ python tests/trachoma_run.py # runs the test model scenarios
```
[the remark this answers is not in the document]
Comment (jeannielynnmoulton) [reply README.md:51]: I'll add this to the sorting out test ticket. It fails for the same reason as the tests.

**Review by tlestang — APPROVED** [review]
Then looks good to me!

File: README.md (hunk 2 above)
Comment (jeannielynnmoulton) [reply README.md:44]: I will merge this PR and consider this comment next week. Thank you!

**Pull request merged**